generators/client/templates/angular/src/main/webapp/app/admin/metrics/_metrics.component.ts

```diff
@@ -31,15 +31,17 @@ export class <%=jhiPrefixCapitalized%>MetricsMonitoringComponent implements OnIn
                 if (key.indexOf('web.rest') !== -1 || key.indexOf('service') !== -1) {
                     this.servicesStats[key] = value;
                 }
-                if (key.indexOf('net.sf.ehcache.Cache') !== -1) {
+            });
+            Object.keys(metrics.gauges).forEach((key) => {
+                if (key.indexOf('jcache.statistics') !== -1) {
+                    let value = metrics.gauges[key].value;
                     // remove gets or puts
                     let index = key.lastIndexOf('.');
                     let newKey = key.substr(0, index);
 
                     // Keep the name of the domain
-                    index = newKey.lastIndexOf('.');
                     this.cachesStats[newKey] = {
-                        'name': newKey.substr(index + 1),
+                        'name': newKey.substr(18),
```

Comment:
btw I would rather do `const JCACHE_KEY = 'jcache.statistics';` and then do `JCACHE_KEY.length` here